style/android/java/install.bat

```diff
@@ -0,0 +1,22 @@
+REM Installs Thoughtbot's IntelliJ configs into your user configs.

```

Comment:
I feel like I have to ask, do we have any windows users at the company?

If not, should we skip this `.bat` file?